packages/server/src/asyncQuestion/asyncQuestion.controller.ts

```diff
@@ -27,6 +27,11 @@ import { AsyncQuestionModel } from './asyncQuestion.entity';
 import { asyncQuestionService } from './asyncQuestion.service';
 import { CourseModel } from 'course/course.entity';
 import { MailService } from 'mail/mail.service';
+
+import OpenAI from 'openai';
```

Comment:
given that we are going to use chatbot api instead of OpenAI directly, we should cleanse related code and packages from office-hours repo